EngineLayer/ProteinParsimony/ProteinGroup.cs

```diff
@@ -143,13 +144,16 @@ public string GetTabSeparatedHeader()
             if (FilesForQuantification != null)
             {
                 bool unfractionated = FilesForQuantification.Select(p => p.Fraction).Distinct().Count() == 1;
-                bool conditionsDefined = FilesForQuantification.All(p => p.Condition == "Default") || FilesForQuantification.All(p => string.IsNullOrWhiteSpace(p.Condition));
+                bool conditionsUndefined = FilesForQuantification.All(p => string.IsNullOrEmpty(p.Condition));
+
+                // this is a hacky way to test for SILAC-labeled data...
```

Comment:
not sure why you'd need to test for SILAC-labeled data here, so the comment doesn't really make sense.

Reply:
I wanted the SILAC reporting to be unaffected by the changes I made to biorep-level summarization. Currently SILAC will report 1 column per label per spectra file, and the SILAC code uses the "condition" field to organize this info, even if the experimental design is not defined by the user. So I needed a way to distinguish between experimental design being used in SILAC automatically vs. being defined by the user

Reply:
I think I understand. Could we name the boolean something that reflects that, if it's only being used for SILAC? Right now the connection isn't intuitive.

Reply:
ya I will update the comment too